dali/pipeline/operators/decoder/nvjpeg_decoder_decoupled_api.h

```diff
@@ -149,6 +150,14 @@ class nvJPEGDecoder : public Operator<MixedBackend>, CachedDecoderImpl {
       const auto file_name = in.GetSourceInfo();
 
       ImageInfo info;
+
+      auto cached_shape = CacheImageShape(file_name);
+      if (volume(cached_shape) > 0) {
+        output_shape_[i] = Dims({cached_shape[0], cached_shape[1], cached_shape[2]});
+        output_info_[i] = info;
```

Comment:
Do we need this reset of `output_info`? I think we can move L152 after this branch

Reply:
will do